src/components/History/HistoryHeader.js

```diff
@@ -0,0 +1,119 @@
+import { useState } from 'react'
+import { Link } from 'react-router-dom'
+import { Header, NavBar, Button, Text, Icon } from '../Reusable'
+import { getTextColor, getIconColor, getBgColor } from '../../utils'
+import { useTheme } from '../../contexts'
+
+const HistoryHeader = () => {
+    const [isSmallNavVisible, setIsSmallNavVisible] = useState(false)
+    const { theme, toggleTheme } = useTheme()
+
+    function toggleNavVisibility() {
+        setIsSmallNavVisible(!isSmallNavVisible)
+    }
+
+    return (
+        <Header>
+
+            <Link to='/'>
+                <Text classes='txt-lg txt-ucase'>
+                    sneakerviews
+                </Text>
+            </Link>
+
+            <NavBar id='nav-big' classes='flx-min-center'>
+
+                <Link to='/videos'>
+                    <Button classes={`btn-txt ${getTextColor(theme)} txt-md txt-lcase mg-right-lg`}>
+                        browse
+                    </Button>
+                </Link>
+
+                <Link to='/playlists'>
+                    <Icon classes={`${getIconColor(theme)} mg-right-lg`}>
+                        playlist_play
+                    </Icon>
+                </Link>
+
+                <Link to='/history'>
+                    <Icon classes={`${getIconColor(theme)} mg-right-lg`}>
+                        history
+                    </Icon>
+                </Link>
+
+                <Link to='/login'>
+                    <Button classes={`btn-txt txt-lcase ${getTextColor(theme)} bg-primary pd-xs txt-md`}>
+                        login
+                    </Button>
+                </Link>
```

Comment:
Nitpicking : This structure here is repeating you can create an array of object and map over it.

Reply:
sure, ill add this suggestion.